condacolab.py

```diff
@@ -338,12 +339,14 @@ def check(prefix: os.PathLike = PREFIX, verbose: bool = True):
     pymaj, pymin = sys.version_info[:2]
     sitepackages = f"{prefix}/lib/python{pymaj}.{pymin}/site-packages"
     assert sitepackages in sys.path, f"💥💔💥 PYTHONPATH was not patched! Value: {sys.path}"
+    assert "/usr/local/" not in sys.path, "💥💔💥 There is some problem with installation!"
     assert (
         f"{prefix}/bin" in os.environ["PATH"]
     ), f"💥💔💥 PATH was not patched! Value: {os.environ['PATH']}"
     assert (
-        f"{prefix}/lib" in os.environ["LD_LIBRARY_PATH"]
-    ), f"💥💔💥 LD_LIBRARY_PATH was not patched! Value: {os.environ['LD_LIBRARY_PATH']}"
+    prefix == os.environ["CONDA_PREFIX"],
+    ), f"💥💔💥 CONDA_PREFIX Value: {os.environ['CONDA_PREFIX']} does not match conda installation location {prefix}!"
```

Comment:
```suggestion
    prefix == os.environ.get("CONDA_PREFIX"),
    ), f"💥💔💥 CONDA_PREFIX Value: {os.environ.get('CONDA_PREFIX')} does not match conda installation location {prefix}!"
```

Use `.get()` to avoid `KeyError` exceptions. If the key is absent, it will return the default value (`None` if not provided as the 2nd argument, which is good enough for us here).